commands/Serve.ts

```diff
@@ -73,9 +82,9 @@ export default class Serve extends BaseCommand {
 
     try {
       if (this.compile === false) {
-        await new BuildWatcher(cwd, this.nodeArgs, this.logger).watch(ADONIS_BUILD_DIR() || './')
+        await new BuildWatcher(cwd, this.nodeArgs, this.logger).watch(ADONIS_BUILD_DIR() || './', this.poll)
```

Comment:
Not really sure if should send them in as-is or make them into configuration objects, like 

```js
.watch({
  buildDir: ADONIS_BUILD_DIR() || './',
  poll: this.poll
})
```

Reply:
I believe it's fine for now, once the arguments will increase, we can pass them as objects